packages/plugin-inline-script-content/inline-script-content.js

```diff
@@ -62,8 +62,11 @@ module.exports = (doc = document, win = window) => ({
 
       const frame = event.errors[0].stacktrace[0]
 
+      // if there are no frames, we can't tell whether this is an inline script
+      if (!frame) return
+
       // if frame.file exists and is not the original location of the page, this can't be an inline script
-      if (frame && frame.file && frame.file.replace(/#.*$/, '') !== originalLocation.replace(/#.*$/, '')) return
+      if (frame && (frame.file && frame.file.replace(/#.*$/, '')) !== originalLocation.replace(/#.*$/, '')) return
```

Comment:
Does this still need the first frame check now?

```suggestion
      if (frame.file && frame.file.replace(/#.*$/, '') !== originalLocation.replace(/#.*$/, '')) return
```